packages/driver/src/cy/multi-domain/index.ts

```diff
@@ -196,12 +207,12 @@ export function addCommands (Commands, Cypress: Cypress.Cypress, cy: Cypress.cy,
                   viewportHeight: Cypress.state('viewportHeight'),
                   runnable: serializeRunnable(Cypress.state('runnable')),
                   duringUserTestExecution: Cypress.state('duringUserTestExecution'),
-                  hookId: state('hookId'),
-                  hasVisitedAboutBlank: state('hasVisitedAboutBlank'),
+                  hookId: Cypress.state('hookId'),
+                  hasVisitedAboutBlank: Cypress.state('hasVisitedAboutBlank'),
```

Comment:
Looks like this was missed when I removed the sync'ing of `about:blank`. Since it's not used anywhere, can you go ahead and remove it?